src/analysis/calculus/tangent_cone.lean

```diff
@@ -317,7 +317,7 @@ begin
   have : _ ⊆ tangent_cone_at 𝕜 (s.prod t) (x, y) :=
     union_subset (subset_tangent_cone_prod_left ht.2) (subset_tangent_cone_prod_right hs.2),
   refine subset.trans _ (closure_mono $ submodule.span_mono this),
-  rw [linear_map.span_inl_union_inr, submodule.prod_coe, closure_prod_eq,
+  erw [linear_map.span_inl_union_inr, submodule.prod_coe, closure_prod_eq,
```

Comment:
Any idea why `erw` is necessary now?

Reply:
It's because in the lemma `subset.trans` is applied to an inequality of `submodule`s, and the unifier returned the goal `closure (λ a, a ∈ ↑S)`, rather than `closure ↑S`, because it's now harder to unify `le` of submodules with `set.subset` I guess.